components/Combobox/index.scss

```diff
@@ -4,6 +4,10 @@
   cursor: pointer;
 }
 
+.hidden {
```

Comment:
not really familiar with this project but here it seems we have already defined .hidden in a components/index file ?
https://github.com/kununu/nukleus/blob/d75f123b1707b8a4a8e444660e0e96748e8142de/components/index.scss#L15